### PR TITLE
Fix Use button when in a subdir

### PR DESCRIPTION
**Pull request by parente**
Form submission path is wrong, needs to be rooted at base_url

### CODE REVIEW

File: nbexamples/static/examples.js [hunk 1]
```diff
@@ -10,14 +10,14 @@ define([
 ], function(Jupyter, $, underscore, utils, dialog) {
```
Comment (ericdill) [inline nbexamples/static/examples.js]: This should be aliased to `_`

Comment (parente) [inline nbexamples/static/examples.js]: Fixed.

**Comment by ericdill** [comment]
lgtm

**Comment by ericdill** [comment]
(says the guy with little js knowledge)

**Pull request merged**